README.md

```diff
@@ -132,17 +85,17 @@ To simulate IOb-SoC's RTL using a Verilog simulator, run
 make sim-run [SIMULATOR=icarus!verilator|xcelium|vcs|questa] [INIT_MEM=0|1] [USE_EXTMEM=0|1]
 ```
 
-The INIT_MEM variable specifies whether the firmware is initially loaded in the memory, skipping the boot process, and the USE_EXTMEM variable indicates whether an external memory such as DRAM is used, in which case the cache system described above is instantiated.
+The INIT_MEM variable specifies whether the firmware is initially loaded in the memory, skipping the boot process, and the USE_EXTMEM variable indicates whether an external memory such as DRAM is used.
 
-The Makefile compiles and runs the software in the `../iob_soc_Vx.y/hardware/simulation` directory. The Makefile includes the `./hardware/simulation/sim_build.mk`, which you can change for your project. To run a simulation test comprising several simulations with different parameters, run
+The Makefile compiles and runs the software in the `../iob_soc_Vx.y/hardware/simulation` directory. The Makefile includes the `../iob_soc_Vx.y/hardware/simulation/sim_build.mk`, which you can change for your project. To run a simulation test comprising several simulations with different parameters, run
 ```Bash
 make sim-test
 ```
 The simulation test contents can be edited in IOb-SoC's top Makefile. 
 
-Each simulator must be described in the [`./hardware/simulation/<simulator>.mk`](https://github.com/IObundle/py2hwsw/tree/main/py2hwsw/hardware/simulation) file. For example, the file `vcs.mk` describes the VCS simulator.
+Each simulator must be described in the [`../iob_soc_Vx.y/hardware/simulation/<simulator>.mk`](https://github.com/IObundle/py2hwsw/tree/main/py2hwsw/hardware/simulation) file. For example, the file `vcs.mk` describes the VCS simulator.
```

Comment:
The supported simulators can be listed with 

py2hwsw sim-ls <=> ls PY2HWSW_ROOT/hardware/simulation/*.mk

bla bla 
py2hwsw sim-cat simulator <=> cat PY2HWSW_ROOT/hardware/simulation/simulator.mk


To support a new simulator or change the makefile segment of  supported simulator place simulator.mk into  iob-soc/hardware/simulation/simulator.mk